.github/workflows/push.yml

```diff
@@ -8,27 +8,27 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - name: Checkout code
-      uses: actions/checkout@v2        
-    
-    - name: Install Node.js
-      uses: actions/setup-node@v1
-      with:
-        node-version: 14.x
-        
-    - uses: actions/cache@v2
-      with:
-        path: '**/node_modules'
-        key: ${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock') }}
-
-    - name: Install packages
-      run: yarn install
-    
-    - name: eslint
-      run: yarn run eslint --ext .js,.ts,.tsx src
-
-    - name: tsc
-      run: yarn run tsc
-
-    - name: prettier
-      run: yarn run prettier --check src
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Install Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14.x
+
+      - uses: actions/cache@v2
+        with:
+          path: '**/node_modules'
+          key: ${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock') }}
+
+      - name: Install packages
+        run: yarn install
+
+      - name: tsc
+        run: yarn run tsc
+
+      - name: eslint
+        run: yarn run eslint --ext .js,.ts,.tsx src
+
+      - name: prettier
+        run: yarn run prettier --check .
```

Comment:
Maybe the separate prettier check is redundant since eslint prettier plugin was installed.

Reply:
@imamatory `prettier` [formats](https://prettier.io/docs/en/index.html) more than just what `eslint` checks  (ie, JSON, HTML, YAML, etc)

Reply:
Ok, then we probably could exclude ts and js files to avoid double-check.